src/streambench/sparkbench/src/multiapi/scala/spark1.3/LatencyCollector.scala

```diff
@@ -64,13 +72,12 @@ class LatencyListener(val ssc:StreamingContext, params:ParamEntity) extends Stre
 
         val avgLatencyAdjust = avgLatency + params.batchInterval.toDouble * 500
         val recordThroughput = params.recordCount / totalTime
-        BenchLogUtil.logMsg("Batch count=" + batchCount)
-        BenchLogUtil.logMsg("Total processing delay=" + totalDelay + "ms")
-        BenchLogUtil.logMsg("Consumed time=" + totalTime + "s")
-        BenchLogUtil.logMsg("Avg latency/batchInterval=" + avgLatencyAdjust + "ms")
-        BenchLogUtil.logMsg("Avg records/sec=" + recordThroughput + "records/s")
-        // ssc.stop(true, true)
-        // System.exit(0)
+        BenchLogUtil.logMsg("Batch count = " + batchCount)
+        BenchLogUtil.logMsg("Total processing delay = " + totalDelay + " ms")
+        BenchLogUtil.logMsg("Consumed time = " + totalTime + " s")
+        BenchLogUtil.logMsg("Avg latency/batchInterval = " + avgLatencyAdjust + " ms")
+        BenchLogUtil.logMsg("Avg records/sec = " + recordThroughput + " records/s")
+        if (!thread.isAlive) thread.start
```

Comment:
Can we avoid output any further logs when the stopping thread has been stared? I noticed there are some latency before the streaming app is stopped so `onBatchCompleted` will still be called. We don't need do anything in this event if the stopping thread has been started.